src/OVAL/probes/probe-api.c

```diff
@@ -1405,7 +1405,7 @@ SEXP_t *probe_item_create(oval_subtype_t item_subtype, probe_elmatr_t *item_attr
         bool    multiplied;
         int     value_i, multiply;
 
-        subtype_name = oval_subtype_to_str(item_subtype);
+        subtype_name = oval_subtype_get_text(item_subtype);
```

Comment:
Always NULL is a good idea, but please check if it isn't in public API. If it is, we shouldn't modify it.

We could also introduce function like this `isValid(str) {return  str != NULL && str!=_invalid}`